packages/spatial/src/transform/components/TransformComponent.ts

```diff
@@ -227,6 +233,15 @@ export const TransformComponent = defineComponent({
     transform.matrixWorld.compose(vec3, quat, scale)
   },
 
+  useInViewingFrustum: () => {
```

Comment:
can we pass in the entity and camera instead? using useEntityContext here is indirectional, and this should be extensible to any camera

Reply:
changed this to just be a function call with params as suggested here. I don't think it being a hook is necessary in this case